build.gradle

```diff
@@ -141,6 +186,12 @@ dependencyUpdates {
 import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 import org.gradle.api.tasks.testing.logging.TestLogEvent
 
+import java.util.regex.Matcher
```

Comment:
Nit: place these imports somewhere **before** their use (I didn't even know Gradle/Groovy allows for importing **after** the use 😯)